src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1224,12 +1224,18 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_ModuleInfoGrouping(Cust
                     .GroupByScriptBlock("Split-Path -Parent $_.Path | ForEach-Object { if([Version]::TryParse((Split-Path $_ -Leaf), [ref]$null)) { Split-Path -Parent $_} else {$_} } | Split-Path -Parent", customControl: sharedControls[0])
                     .AddHeader(Alignment.Left, width: 10)
                     .AddHeader(Alignment.Left, width: 10)
+                    .AddHeader(Alignment.Left, label: "PreRelease", width: 10)
```

Comment:
Instead of adding a new column, why not make the Version column wider and append the prerelease tag so it looks exactly like it does in the PowerShell gallery?

Reply:
The only concern is that the `Version` property doesn't reflect what you see on the screen.